### PR TITLE
[ADD] event_badge_design

### PR DESCRIPTION
**Pull request by hbrunn**

### CODE REVIEW

**Review by petrus-v — APPROVED** [review]
LGTM code review + test on runboat !

File: event_badge_design/views/event_event.xml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+<!-- Copyright 2022 Hunki Enterprises BV
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+
+<record id="view_event_form" model="ir.ui.view">
+    <field name="model">event.event</field>
+    <field name="inherit_id" ref="event.view_event_form" />
+    <field name="arch" type="xml">
+        <notebook position="inside">
+            <page name="badge_front" string="Badge front">
```
Comment (gdgellatly) [inline event_badge_design/views/event_event.xml:14]: Missing field event_logo?

**Review by gdgellatly — CHANGES_REQUESTED** [review]
Hi, Sorry real quick review. Really appreciate this.

File: event_badge_design/views/event_event.xml (hunk 1 above)
Comment (hbrunn) [reply event_badge_design/views/event_event.xml:14]: ah right, added it on its own tab

**Review by rafaelbn — APPROVED** [review]
Tested un runboat 👍 

imagen

**Comment by rafaelbn** [comment]
Please @gdgellatly update your review! 😄 Thank you!

**Review by gdgellatly — APPROVED** [review]

**Review by primes2h — APPROVED** [review]
LGTM.

Thanks!

**Pull request merged**